cmd/regex_format.go

```diff
@@ -198,6 +185,11 @@ func processFile(filePath string, ctxt *processors.Context, checkOnly bool) erro
 		lines = append(lines, string(line))
 	}
 
+	if checkStandardHeader(lines) {
+		logger.Error().Msgf("file %s does not have standard header", filePath)
```

Comment:
Log at info level? You're fixing it on the next line...

Reply:
Makes sense. Fixed.